test/execution-tests/3.0.1_resolveModuleName/test/app.tests.ts

```diff
@@ -0,0 +1,12 @@
+import * as app from '../src/app';
+
+// We don't actually care about the result of the following operations;
+// what we care about is typescript resolving json as modules
```

Comment:
Could you amend the comment to reflect what the intention of the test is please?

Reply:
I left the comment because it's actually the same intent - we don't care about what is being built, just about TS being able to make the resolution

Reply:
Oh, I see a "json" is in the comment - will change it 👍